grunttasks/jscs.js

```diff
@@ -6,13 +6,23 @@ module.exports = function (grunt) {
   'use strict';
 
   grunt.config('jscs', {
-    src: [
-      '{,<%= yeoman.app %>/**/,tests/**/,grunttasks/**/,server/**/,scripts/**/}*.js',
-      '!<%= yeoman.app %>/bower_components/**',
-      '!<%= yeoman.app %>/scripts/vendor/**'
-    ],
-    options: {
-      config: '.jscsrc'
+    server: {
+      src: '{,tests/**/,grunttasks/**/,server/**/,scripts/**/}*.js',
+      options: {
+        config: '.jscsrc'
+      }
+    },
+    app: {
+      src: [
+        '{,<%= yeoman.app %>/**/,tests/**/,grunttasks/**/,server/**/,scripts/**/}*.js',
```

Comment:
Something seems funny here. If I'm reading this correctly, we're linting the root *.js files twice (one in each target &mdash; note that leading `{,` bit on both lines 10 and 17).

But I'd have to clone this and run `grunt jscs --verbose` to confirm.